cothink2/client/itemmanager.js

```diff
@@ -0,0 +1,14 @@
+itemManager = (function () {
+
+    var getItemById = function (id) {
```

Comment:
When you use this construct to provide an unexposed object-wide scope, you could use named functions, preference thing I guess.